content/source/docs/enterprise/admin/update-tfe-license.html.md

```diff
@@ -0,0 +1,161 @@
+
+### Manual Updates
+
+#### Replicated Console
+
+1.  Navigate to the Replicated console, which is located at `https://<<TFE_HOSTNAME>>:8800`, click on the gear icon on the top-right corner, and select the **View License** menu.
+
+    ![View License](./images/view-license.png)
+
+2.  Click on **Sync License** to start the operation.
+
+    ![Sync License](./images/sync-license.png)
+
+#### Replicated Command Line Interface
+
+1.  Obtain the newly updated license file from HashiCorp and copy it to the Terraform Enterprise host machine.
+
+2.  Connect to the Terraform Enterprise host machine using SSH.
+
+3.  Use the following command to load the new license.
+
+        $ replicatedctl license-load < /path/to/license.rli
+
+## Update License - Airgap Installation Type
+
+### Replicated Console
+
+1.  Navigate to the Replicated console, which is located at `https://<<TFE_HOSTNAME>>:8800`.
+
+2.  Click on the gear icon on the top-right corner and select the **Console Settings** menu.
+
+    ![Console Settings](./images/console-settings.png)
+
+3.  Select **Airgapped Settings** from the left-hand menu and upload the license.
+
+    ![Upload license](./images/upload-license.png)
+
+### Replicated Command Line Interface
+
+1.  Obtain the updated license and airgap package from HashiCorp and copy them to the Terraform Enterprise host.
+
+2.  Connect to the Terraform Enterprise host machine using SSH.
+
+3.  Execute the following command, updating the paths as needed.
+
+        $ replicatedctl license-load --airgap-package /path/to/bundle.airgap < /path/to/license.rli
+
+## Troubleshooting Guides
+
+### No error but the license is not updated
+
+In the Airgap installation, the license update operation may finish without any error but the license detail displayed on the Replicated console or the Replicated command line output is still not updated. This symptom can be related to the incorrect license file being used. In order to further identify the symptom, please follow these steps:
+
+1.  Connect to the Terraform Enterprise host machine using SSH.
+
+2.  Examine the output of the command `docker logs replicated` and look for the warning message below.
+
+         WARN 2021-02-22T01:40:00+00:00 tasks/app_tasksteps.go:113 Airgap license on disk does not match installed license
+
+3.  If this warning message occurs, it indicates that the incorrect license file is being used. Please contact your Customer Success Manager if you have one assigned to your account, or contact [HashiCorp Support](https://www.hashicorp.com/technical-support-services-and-policies) for further assistance. When contacting support, please include the output from the `replicatedctl license inspect` command after running it on the existing Terraform Enterprise host machine.
+
+### Unable to sync license: Error: Unsuccessful HTTP response
+
+The error message below may occur due to several possible issues.
+
+    Unable to sync license: Error: Unsuccessful HTTP response
```

Comment:
Wrapped in code fence

````suggestion
    ```plaintext
    Unable to sync license: Error: Unsuccessful HTTP response
    ```
````